horovod/common/message.cc

```diff
@@ -14,65 +14,73 @@
 // limitations under the License.
 // =============================================================================
 
-#include "mpi_message.h"
-#include "wire/mpi_message_generated.h"
+#include "message.h"
+
 #include <iostream>
 
+#include "wire/mpi_message_generated.h"
+
 namespace horovod {
 namespace common {
 
-const std::string& MPIDataType_Name(MPIDataType value) {
+const std::string& DataType_Name(DataType value) {
   switch (value) {
-  case HOROVOD_UINT8:
-    static const std::string uint8("uint8");
-    return uint8;
-  case HOROVOD_INT8:
-    static const std::string int8("int8");
-    return int8;
-  case HOROVOD_UINT16:
-    static const std::string uint16("uint16");
-    return uint16;
-  case HOROVOD_INT16:
-    static const std::string int16("int16");
-    return int16;
-  case HOROVOD_INT32:
-    static const std::string int32("int32");
-    return int32;
-  case HOROVOD_INT64:
-    static const std::string int64("int64");
-    return int64;
-  case HOROVOD_FLOAT16:
-    static const std::string float16("float16");
-    return float16;
-  case HOROVOD_FLOAT32:
-    static const std::string float32("float32");
-    return float32;
-  case HOROVOD_FLOAT64:
-    static const std::string float64("float64");
-    return float64;
-  case HOROVOD_BOOL:
-    static const std::string bool_("bool");
-    return bool_;
-  default:
-    static const std::string unknown("<unknown>");
-    return unknown;
+    case HOROVOD_UINT8:
+      static const std::string uint8("uint8");
+      return uint8;
+    case HOROVOD_INT8:
+      static const std::string int8("int8");
+      return int8;
+    case HOROVOD_UINT16:
+      static const std::string uint16("uint16");
+      return uint16;
+    case HOROVOD_INT16:
+      static const std::string int16("int16");
+      return int16;
+    case HOROVOD_INT32:
+      static const std::string int32("int32");
+      return int32;
+    case HOROVOD_INT64:
+      static const std::string int64("int64");
+      return int64;
+    case HOROVOD_FLOAT16:
+      static const std::string float16("float16");
+      return float16;
+    case HOROVOD_FLOAT32:
+      static const std::string float32("float32");
+      return float32;
+    case HOROVOD_FLOAT64:
+      static const std::string float64("float64");
+      return float64;
+    case HOROVOD_BOOL:
+      static const std::string bool_("bool");
+      return bool_;
+    case HOROVOD_BYTE:
+      static const std::string byte_("byte");
+      return byte_;
+    case HOROVOD_NULL:
+      static const std::string null_("null");
+      return null_;
+    default:
+      static const std::string unknown("<unknown>");
+      return unknown;
   }
 }
 
 const std::string& MPIRequest::RequestType_Name(RequestType value) {
```

Comment:
This can be renamed from `MPIRequest` to simply Request or HorovodRequest.  There should not be any MPI specifics.

Reply:
Done in #841.